### ISSUE TITLE
Training taking a long time: Hyperparameters unoptimized?

### ISSUE THREAD

**Issue by Peterard** [Unity-Technologies/ml-agents#964]
Sorry if this is a duplication from somewhere, I checked but I couldn't find anything that answers my problem.

I have made a super simple adversarial AI game whereby 2 circles, a "chaser" and an "escapee" learn to move towards or away from each other respectively. Each AgentAction, a reward is added which is simply the distance between the two entities multiplied by a constant (or a constant minus the normalized distance to keep all rewards between 0 and 1).

I have two problems:
* The training takes an extraordinarily long time (roughly 50-100K steps or roughly 20-30 mins) to give the two agents anything close to the desired behavior (i.e. the chaser actually moving towards the escapee and vice versa). Since this is such a simple task, I would assume that there is a much faster way to train these agents. I have seen implementations of ML on "slow" technologies such as synaptic.js which can train these sort of tasks an order of magnitude or two faster than this, so I assume that I am making a mistake somewhere.
* Once the agents learn to chase and escape, they always just pick 2 corners and run into them. There is (seemingly) no desire to try to move to the other corners of the game arena, which makes me question why there is not more evolution in the agents strategy as time passes. I have tried setting the beta hyperparameter as high as recommended, but not avail.

My hyperparameters are default except for:
```
    max_steps: 9.0e5
    learning_rate: 5.0e-4
    beta: 1.0e-2
```
For both chaser and escapee.

I have saved the following file in the "Examples" folder of my ml-agents unity environment, and run the training from there. Feel free to try it out for yourself:
https://github.com/Peterard/ml_agents_chaser

Where do you think this could be going wrong?

**Comment by awjuliani** [comment]
Hi @Peterard,

Two hyperparameters you can adjust to increase training speed are `learning_rate` and `num_epochs`. The former could possibly be increased up to `1e-3` and the latter up to `10`.

**Comment by eshvk** [comment]
Thanks for reaching out to us. Hopefully you were able to resolve your issue.  We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.
